docs/home/index.md

```diff
@@ -12,28 +12,33 @@ orphan: true
 
 # CrateDB Documentation
 
-Welcome to the official CrateDB Documentation. Whether you're a developer,
+Welcome to the official CrateDB Documentation. Whether you are a developer,
 database administrator, or just starting your journey with CrateDB, our
 documentation provides the information and knowledge needed to build
 real-time analytics and hybrid search applications that leverage CrateDB's
 unique features.
 
+:::{rubric} Benefits
+:::
+* In a unified data platform approach, CrateDB includes analyzing relational, JSON,
+  time-series, geospatial, full-text, and vector data within a single system,
+  eliminating the need for multiple databases.
+* The fully distributed SQL query engine, built on top of Apache Lucene,
```

Comment:
```suggestion
* The fully distributed query engine
```

Reply:
We are doing that name dropping on purpose, because CrateDB did not mention them at all for a very long time, thus not being able to build upon their fame. I actively lobbied for that in the past, and other people strongly agreed.

Reply:
Possible improvement: We could mention the technical foundation within a _separate_ enumeration item.

Reply:
As a further example to that rationale, see also the [canonical feature page about vector search](https://cratedb.com/docs/guide/feature/search/vector/), where we are very explicit about the implementation details within, well, a relevant "Details" section.

> ### Details
>
> CrateDB uses Lucene as a storage layer, so it inherits the implementation and concepts of Lucene Vector Search, in the same spirit as Elasticsearch.
>
> To learn more details about what’s inside, please refer to the [HNSW](https://en.wikipedia.org/wiki/Hierarchical_navigable_small_world) graph search algorithm, [how Lucene implemented it](https://www.apachecon.com/acna2022/slides/04_lucene_vector_search_sokolov.pdf), [how Elasticsearch now also builds on it](https://www.elastic.co/search-labs/blog/articles/vector-search-elasticsearch-rationale), and why effectively [Lucene Is All You Need](https://arxiv.org/pdf/2308.14963).
>
> While Elasticsearch uses a [query DSL based on JSON](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html), in CrateDB, you can work with Lucene Vector Search using SQL.

The idea is to very explicitly convey you are getting the best of both worlds, a solid foundation approved by the whole industry FOSS and not, and a sweet SQL interface on top, what others can't provide.